www.foia.gov/_sass/modules/_hero_unit.scss

```diff
@@ -0,0 +1,78 @@
+.usa-hero {
+  background-image: none;
+  padding: 0;
+  @media screen and (min-width: $small-screen) {
+    background-image: url('../assets/Usdepartmentofjustice-1.jpg');
+    padding-top: 3rem;
+    padding-bottom: 3rem;
+  }
+  @media screen and (max-width: $small-screen) {
+    .usa-grid {
+      padding: 0;
+    }
+  }
+}
+.usa-hero-callout {
+  min-width: 100%;
+  @media screen and (min-width: $medium-large-screen){
+    min-width: 55%;
+  }
+  h2 {
+    font-size: 2.2rem;
+    font-family: $font-serif;
+    font-weight: bold;
+    line-height: 1.5;
+
+    @media screen and (min-width: $small-screen) {
+      font-size: 2.5rem;
```

Comment:
Should this be one of the mobile size variables?

Reply:
I think no, though I can make it a variable to the local file. The type treatment in the hero unit is unique, so the styles applicable there won't really be of much use in the global sense.